src/components/Stack/Stack.tsx

```diff
@@ -3,24 +3,32 @@ import { forwardRef } from 'react';
 import { Space } from '~/components/Space/Space';
 import { Box } from '~/components/Box/Box';
 import { Children } from '~/utils/typing/children';
-import { SPACE_UNIT } from '~/utils/constants';
 
-export type StackProps = ComponentProps<typeof Space> &
-  ComponentProps<typeof Box> & {
+export type StackProps = Omit<ComponentProps<typeof Space>, 'size' | 'fluid'> &
```

Comment:
~ I will strongly prefer to decouple the `StackProps` type definition from the other types definition (`Space` and `Box`). Despite of the more verbosity needed, every layout component should be used (and maybe exported in a future) as an independent component

Reply:
I get your point but since `Stack` is built upon `Box` I would likely remove the `Space` props but keep the `Box` to prevent misalignment and repetitions.